src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -1375,7 +1386,11 @@ private void RunParserReliably(BulkCopySimpleResultSet bulkCopyHandler = null)
             internalConnection.ThreadHasParserLockForClose = true;
             try
             {
-                _parser.RunReliably(RunBehavior.UntilDone, null, null, bulkCopyHandler, _stateObj);
+                #if NETFRAMEWORK
```

Comment:
I'm not 100% sure the implications of using one versus the other.

Reply:
It's purely CER-related, if the method existed in netcore in its current form then there'd be no side-effects to calling it on both targets.